_docs/_latest/configure/instrumentation/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Instrumentation 
+order: 200
+---
+
+# {{ page.title }}
+
+OpenCensus tracing has been added to Forseti as part of instrumentation on 
+Forseti. 
+
+---
+
+## Configuring Instrumentation
+
+**Tracing**
+* Tracing is enabled by default and the traces are automatically 
+exported to `Stackdriver Trace`.
+* If you wish to disable tracing, you may do so by modifying the 
+`initialize_forseti_services.sh`script found under `install/gcp/scripts` to 
+remove `FORSETI_COMMAND+=" --enable tracing"` and redeploy Forseti. 
```

Comment:
Hmmm, the way that `--enable-tracing` is flag is placed in the `install/gcp/scripts/initialize_forseti_services.sh`, it will not be configurable.  i.e meaning that everytime the VM is restarted it will pull the file from github again and write over any changes here.

If you really want to make it configurable, you would have to set this up in the `forseti_server_conf.yaml` file and the application would determine if it should run or not.